src/NexusMods.DataModel/ApplyService.cs

```diff
@@ -12,14 +15,19 @@ public class ApplyService : IApplyService
 {
     private readonly ILoadoutRegistry _loadoutRegistry;
     private readonly ILogger<ApplyService> _logger;
+    private readonly IDiskStateRegistry _diskStateRegistry;
+    private readonly IObservable<(GameInstallation gameInstallation, IId loadoutRevision)> _lastAppliedRevisionObservable;
 
     /// <summary>
     /// DI Constructor
     /// </summary>
-    public ApplyService(ILoadoutRegistry loadoutRegistry, ILogger<ApplyService> logger)
+    public ApplyService(ILoadoutRegistry loadoutRegistry, IDiskStateRegistry diskStateRegistry, ILogger<ApplyService> logger)
     {
         _loadoutRegistry = loadoutRegistry;
         _logger = logger;
+        _diskStateRegistry = diskStateRegistry;
+
+        _lastAppliedRevisionObservable = _diskStateRegistry.LastAppliedRevisionObservable;
```

Comment:
`ApplyService` shouldn't have the observable property from `IDiskStateRegistry` as a private field.